slices/x11-common.yaml

```diff
@@ -0,0 +1,33 @@
+package: x11-common
+
+essential:
+  - x11-common_copyright
+
+slices:
+  resources:
+    contents:
+      /etc/X11/Xresources/x11-common:
+
+  scripts:
```

Comment:
could these be tested too?

Reply:
@cjdcordeiro scripts do depend on #!/bin/sh to be present. Should I add a mutation script for this?

Reply:
Xreset requires (implicitly) debianutils (specifically `run-parts`). Should I slice it too so we can test it properly?

Reply:
I'd say so. Or drop the slice. The general rule is that we should only create the slices we need. I.e., even if this package has these files, if you don't need them, don't slice them (cause you won't be able to test them properly and thus might be offering a slice that is broken)

Reply:
I will have a look at debian utils first. Despite their usage is implicit, it might be worth to at least mention that, though it is pretty sad that package doesn't have a direct dependency for it, so it is explicit.

Reply:
Had a look at the slice. It is easier to drop it (given `scripts` is for a complete package slicing) rather than to bring full wm to test Xsession script.

Reply:
Got it. I'll unresolve this until it's dropped. Pls let me know once it's ready for another review ;)